src/applications/gi/sass/gi.scss

```diff
@@ -1,15 +1,15 @@
-@import '~@department-of-veterans-affairs/formation/sass/shared-variables';
-@import '~@department-of-veterans-affairs/formation/sass/modules/m-modal';
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```

Comment:
looks like linting change?

Reply:
Yes, everything other than the addition of the `disabled` class in `gi.scss` was due to linting.

Reply:
I think the single quote is preferred.

Reply:
Pushed a change that uses the single quote instead for the imports.